packages/govuk-frontend/src/govuk/components/service-navigation/service-navigation.puppeteer.test.js

```diff
@@ -229,5 +249,97 @@ describe('/components/service-navigation', () => {
         expect(toggleExpandedAttribute).toBe('false')
       })
     })
+
+    describe('on wide viewports', () => {
+      beforeAll(async () => {
+        // First let's reset the viewport to a desktop one
+        await page.setViewport({
+          width: 1280,
+          height: 720
+        })
+      })
+
+      afterAll(async () => {
+        // After tests have run reset teh viewport to a phone
+        await page.emulate(iPhone)
+      })
+
+      describe('on page load', () => {
```

Comment:
Depending on the outcome of the other thread, we'd also want a test checking for `disabled` here as well